auth.rb

```diff
@@ -1,14 +1,10 @@
 class Authentication
-  # please improve this rails action,
-  # if you'd like to put some code in another file please state so:
-  # authentication API
   def auth
-    user = User.find(params[:username])
-    if user.check_password(params[:password])

```

Comment:
Needs to handle `user.present?` whether user is nil or not. It will prevent from `NoMethodError: undefined method `check_password' for nil:NilClass`